package io.novafoundation.nova.feature_account_impl.domain.errors

class UnsupportedNetworkException : RuntimeException()
